integrations/pjrt/README.md

```diff
@@ -78,14 +77,7 @@ The plugin `openxla_pjrt_artifacts` is in the `ctstools` directory and
 performs additional manipulation of the environment in order to save
 compilation artifacts, reproducers, etc.
 
-## Contacts
-
-* [GitHub issues](https://github.com/openxla/openxla-pjrt-plugin/issues):
-  Feature requests, bugs, and other work tracking
```

Comment:
Now a 404 page.

Reply:
Latest upstream: https://github.com/openxla/xla/tree/main/xla/pjrt/c#communication-channel

```markdown
## Communication channel

*   Please file issues in the [OpenXla/xla repo](https://github.com/openxla/xla).
*   Join the [pjrt-announcement maillist](https://groups.google.com/g/pjrt-announce/).
```

Could link to that first github link here, though for topics relating to IREE's implementation of the API we'll want to keep discussions on this github repo or IREE's Discord server (`#jax` or `#pjrt-plugin` channels)

Reply:
Good to know. I've added the IREE issues page and the Discord server.